lessons/projects/5.mdx

```diff
@@ -0,0 +1,1379 @@
+                width='200px'
+                height='200px' />
+              <button
+                value='0.02'
+                onClick={(e) => mintToken(e)}
+                style={NFTMint}
+                disabled={isMintLoading}>
+                Mint
+              </button>
+            </div>
+            <div style={NFTCard} >
+              Tier 2
+              <Image
+                src='/nfts/2_premium.svg'
+                width='200px'
+                height='200px' />
+              <button
+                value='0.05'
+                onClick={(e) => mintToken(e)}
+                style={NFTMint}
+                disabled={isMintLoading}>
+                Mint
+              </button>
+            </div>
+          </div>
+        </main>
+
+// rest of the code...
+```
+
+Let's see what is happening in each NFT Card by taking `Tier 0` as an 
+example.
+
+```jsx
+<div style={NFTCard}>
+  Tier 0
+  <Image
+    src='/nfts/0_basic.svg'
+    width='200px'
+    height='200px' />
+  <button
+    value='0.01'
+    onClick={(e) => mintToken(e)}
+    style={NFTMint}
+    disabled={isMintLoading}>
+    Mint
+  </button>
+</div>
+```
+
+- We start off by displaying the NFT Tier name.
+- A nextJS `Image` element renders a preview of the NFT Image. Earlier,
+we saved our images in the `/public/nfts` folder. A nextJS app
+automatically detects content from the `/public` directory so we can
+directly reference the source as `/nfts/0_basic.svg`. We also need 
+to pass in the width and height explicitly.
+- A `button` element let's us call the `mintToken` function `onClick` and 
+we send the value associated with the button to that function so that 
+it can be used to mint the specific tier of NFT. Make sure you pass 
+in the right value for each tier. Lastly, we want to disable the 
+button when another mint is underway. For this we use the true/ false 
+values received from `isMintLoading`. The button will be disabled when 
+`isMintLoading` is `true`.
+
+Great but anyone can view and call our mint functions even if their 
+wallet is not connected to the application. This could result in some 
+errors. We must check if a user is connected before we let them 
+interact with our application simply by using the `isConnected` utility
+from Wagmi and setting setting its value in our `isUserConnected` state.
+
+```jsx
+// imports and consts...
+
+  useEffect(() => {
+    try {
+      setIsUserConnected(isConnected);
+    } catch (error) {
+      console.log("Error connecting to user", error.message);
+    }
+  }, [isConnected])
+
+// function Home code...
+```
+
+The `useEffect` hook runs the functions within it everytime the value of 
+its dependencies (`isConnected`, in this case) changes.
+
+Then we can wrap the `<main>` element as follows:
+
+```jsx
+// return statement for rendering...
+
+      {isUserConnected ?
+        <main className={styles.main}>
+          
+          {/* main code here... */}
```

Comment:
`/* main code here... */}` try a more explicit description here - maybe show the opening and closiing  tags of the element it represents.
I know you are trying to use a convention, where you are not using the actual lesson body text to describe what is  in these *spaces*, and I take my hat  off to your being super succinct, but for me as a user, it is absolutely arduous (as in hours and hours), trying to figure out what these descriptions mean sometimes. I would suggest, use this opportunity to use more words/characters to describe what you mean to represent. And try and learn from there how to be more succinct in future lessons you write. Once again, hats off, but it's very tiring for me as a *learner*.